internal/provider/kubernetes/gateway.go

```diff
@@ -377,7 +377,12 @@ func (r *gatewayReconciler) Reconcile(ctx context.Context, request reconcile.Req
 	if !found {
 		gw, ok := r.resources.Gateways.Load(request.NamespacedName)
 		if !ok {
-			r.log.Info("failed to find gateway in the watchable map", "namespace", gw.Namespace, "name", gw.Name)
```

Comment:
uggh 🙈 , can you just print `request.NamespacedName.String` ?

Reply:
Yep